test/cmd/discovery.sh

```diff
@@ -55,7 +55,7 @@ run_assert_short_name_tests() {
   output_message=$(kubectl get --raw=/api/v1)
 
   ## test if a short name is exported during discovery
-  kube::test::if_has_string "${output_message}" '{"name":"configmaps","singularName":"","namespaced":true,"kind":"ConfigMap","verbs":\["create","delete","deletecollection","get","list","patch","update","watch"\],"shortNames":\["cm"\]}'
+  kube::test::if_has_string "${output_message}" '{"name":"configmaps","singularName":"","namespaced":true,"kind":"ConfigMap","verbs":\["create","delete","deletecollection","get","list","patch","update","watch"\],"shortNames":\["cm"\]'
```

Comment:
This change is needed because the `output_message` does not end at "shortNames", it also has "storageVersionHash" now.

Reply:
Please add this in a comment, or add a `,"storageVersionHash":`?

Reply:
Added a `,"storageVersionHash":`